lib/galaxy/config/sample/galaxy_vscode.ini

```diff
@@ -0,0 +1,1434 @@
+#
```

Comment:
Can you remove all the commented out stuff? If you're using this you hopefully know what options to set.